lhotse/recipes/aishell2.py

```diff
@@ -40,7 +40,7 @@ def prepare_aishell2(
     manifests = defaultdict(dict)
     dataset_parts = ["train"]
     for part in tqdm(
-        dataset_parts, desc="process aishell2 audio, it needs waste about 6 hours time."
+        dataset_parts, desc="Process aishell2 audio, it takes about 6 hours."
```

Comment:
I think 6 hours are too long for a 1000 hours dataset.

Reply:
Yes, I agree with you. I will implement it.